src/Receipts/ReceiptResponse.php

```diff
@@ -1,90 +1,110 @@
 <?php
 
-
 namespace Imdhemy\AppStore\Receipts;
 
-use Imdhemy\AppStore\Exceptions\InvalidReceiptException;
+use Imdhemy\AppStore\ValueObjects\LatestReceiptInfo;
 use Imdhemy\AppStore\ValueObjects\PendingRenewal;
 use Imdhemy\AppStore\ValueObjects\Receipt;
-use Imdhemy\AppStore\ValueObjects\ReceiptInfo;
 use Imdhemy\AppStore\ValueObjects\Status;
 
 /**
  * Class ReceiptResponse
  * @package Imdhemy\AppStore\Receipts
+ * @see https://developer.apple.com/documentation/appstorereceipts/responsebody
  */
 class ReceiptResponse
 {
+    public const ENV_SANDBOX = 'Sandbox';
+    public const ENV_PRODUCTION = 'Production';
+
     /**
-     * @var string
+     * The environment for which the receipt was generated.
+     * @var string|null
      */
     protected $environment;
```

Comment:
since we use native return types already, I'd use typed properties from 7.4 here as well: ?string

Reply:
Good idea. Unfortunately, I followed the same convention in all other files. It would be annoying to update them all now 😄. I'll consider this next time.